### ISSUE TITLE
Your best result doesn't come from the same weight file

### ISSUE THREAD

**Issue by jinzhengbei** [tpnam0901/QTSeg#13]
Hello, I found that your best result doesn't come from the same weight file. Can such a result be included in the paper?

**Comment by tpnam0901** [comment]
Hello @jinzhengbei, 

We used the same weight file to report all results in our paper. Although multiple checkpoints were saved during training, only weight_best_iou.pt was used for evaluation. I think you might be confusion because of we save many checkpoints. For example, this is the evaluation from weight_best_iou.pt checkpoint. 

<img width="1140" height="63" alt="Image" src="https://github.com/user-attachments/assets/950a79fb-e3ac-4d9d-9ce4-394430fefa7a" />

I don't know if I understand your question correctly, but if there's any ambiguity or further clarification needed, feel free to let me know.

**Comment by jinzhengbei** [comment]
Thank you for your explanation. I understand this. 

Regarding the DSB2018 dataset, I noticed that QTSeg/src/tools/dsb2018_01_combine_mask.py only processed the training set stage1_train and did not use stage1_test. Could you please clarify if stage1_test was used during the validation process?

**Comment by tpnam0901** [comment]
We don't have the label for the stage1_test, so the stage1_test was not used during the validation process. Rather than, we randomly split the stage1_train into five folds using dsb2018_02_split_folds.py

**Comment by jinzhengbei** [comment]
Thank u@tpnam0901. Thanks for your answer!
